runtime/help/commands.md

```diff
@@ -71,9 +71,20 @@ quotes here but these are not necessary when entering the command in micro.
 
 * `setlocal 'option' 'value'`: sets the option to value locally (only in the
    current buffer). This will *not* modify `settings.json`.
+   
+* `toggle 'option'`: toggles the option true/false. Only works with boolean options.
+   This will modify your `settings.json` with the new value.
+    
+* `togglelocal 'option'`: toggles the option true/false locally (only in the current buffer). 
```

Comment:
This line is a bit too long, please split (and perhaps line 75 too). As you may notice, we try to keep the maximum width of lines in the documentation to be under 80 characters (well, sometimes just a bit above 80 characters).